src/kmeans.jl

```diff
@@ -17,12 +17,14 @@ const _kmeans_default_init = :kmpp
 const _kmeans_default_maxiter = 100
 const _kmeans_default_tol = 1.0e-6
 const _kmeans_default_display = :none
+const _kmeans_default_n_init = 10
```

Comment:
10 was chosen as this is the default for sklearn n_init param, and since we are duplicating this functionality, I just took this value.